mapsforge-core/src/main/java/org/mapsforge/core/util/Parameters.java

```diff
@@ -64,6 +69,8 @@ public enum SymbolScaling {ALL, POI}
      */
     public static int NUMBER_OF_THREADS = Runtime.getRuntime().availableProcessors() + 1;
 
+    public static final long MAX_MEMORY_MB = Runtime.getRuntime().maxMemory() / 1000 / 1000;
```

Comment:
@Sublimis This is not a parameter, but a calculated constant. It should be moved to the class that uses it.

Also, shouldn't the calculation be done with `1024'?

Reply:
What if more than one class wants to use a constant, shouldn't there be some class holding constants?
Can we create it?
https://en.wikipedia.org/wiki/Megabyte

Reply:
@Sublimis `Parameters` is not the class to hold constants.
We can create a `Constants` class for such purpose.